.github/workflows/renovate.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           sudo chown -R runneradmin:root /tmp/renovate/
           ls -R "$cache_dir"
 
-      - uses: renovatebot/github-action@v41.0.5
+      - uses: renovatebot/github-action@v41.0.7
```

Comment:
Same.

```suggestion
      - uses: renovatebot/github-action@v41
```

Reply:
There is no `v41` tag in `github-actions`, because of the [maintainers' decision](https://github.com/renovatebot/github-action/issues/791). We even have a link to our repository in that issue 😆 , because our GH actions failed when we tried to use major-only version.